gcp/upload_bcl_gcp

```diff
@@ -21,18 +21,24 @@ trap exit_handler EXIT
 
 while true; do
     find "${flowcells_dir}" -mindepth 1 -maxdepth 1 -type d -not -name "TestRuns" -not -name "MyRun" | while read -r flowcell_path; do
-        echo "Starting BCL upload of ${flowcell_path}"
-        gs_path="gs://${bucket}/$(basename $flowcell_path)"
-        gcloud auth activate-service-account --key-file "${gcp_cred}"
-        rsync_bcls $flowcell_path $gs_path
+        upload_complete="${flowcell_path}/.gcp_upload_complete"
```

Comment:
Maybe instead of `.gcp_upload_complete` nicer to call the file `GCPUploadComplete.txt` so it aligns with RTAComplete format and is not hidden?